cpp/net/url_fetcher_test.cc

```diff
@@ -7,6 +9,9 @@
 #include <sys/stat.h>
 #include <sys/wait.h>
 #include <unistd.h>
+#ifdef HAVE_VFORK_H
```

Comment:
Since there's a separate `HAVE_VFORK_H` and `HAVE_WORKING_VFORK`, I wonder where `vfork()` is when you don't have `vfork.h`?!? Or maybe there's never a working `vfork()` when there isn't a `vfork.h`?

Anyway, works on Linux and Mac OS X, good enough for me. :wink: